apps/explorer/src/components/ownedobjects/views/OwnedObjectView.tsx

```diff
@@ -37,7 +38,9 @@ export default function OwnedObjectView({ results }: { results: DataType }) {
                     <div className={styles.ownedobjectheader}>
                         <h2>Coins</h2>
                     </div>
-                    <OwnedCoinView results={coin_results} />
+                    <ErrorBoundary>
```

Comment:
Feels like something that more logically should exist in the composing page

Reply:
Done